src/components/AddItemForm/AddItemForm.js

```diff
@@ -0,0 +1,115 @@
+import { useState } from 'react';
+import { db } from '../../lib/firebase.js';
+
+const AddItemForm = ({ listId }) => {
+  const defaultFormValues = {
+    itemName: '',
+    purchaseInterval: '7',
+  };
+
+  const [formValues, setFormValues] = useState(defaultFormValues);
+
+  // generic function updates formValues state for any of the below form inputs
+  const handleChange = (event) => {
+    const inputName = event.target.name;
+    setFormValues({ ...formValues, [inputName]: event.target.value });
+  };
+
+  // on form submit, add user input and other default values to database
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    const newItem = formValues;
+    newItem.purchaseInterval = parseInt(newItem.purchaseInterval); // convert purchaseInterval to integer
+    newItem.lastPurchaseDate = null;
+    newItem.numberOfPurchases = 0;
+
+    try {
+      await db.collection(`lists/${listId}/items`).add(formValues); // add item to Firestore database
+      setFormValues(defaultFormValues); // after saving to database, reset form values to defaults
+    } catch (err) {
+      console.log(err);
+    }
+  };
```

Comment:
Would be good to handle giving feedback to the user upon success/failure in here as well.